README.md

```diff
@@ -213,11 +213,14 @@ sublime-phpcs is insanely powerful, but if you'd prefer automatic linting, [Subl
 - Install [linter-phpcs](https://atom.io/packages/linter-phpcs) via Atom's package manager.
 - Run `which phpcs` to get your `phpcs` executable path.
 - Enter your `phpcs` executable path and one of the coding standards specified above (e.g. `WordPress`, `WordPress-VIP`, etc.).
+- Change the Tab Width setting to `4`.
```

Comment:
Agreed, see https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1086/commits/1a18cfd3bb31809c9a64eaa7e23ab73786e3e6c9

Frankly the instructions as they were weren't completely clear, hopefully this helps :)